Makefile

```diff
@@ -5,7 +5,7 @@ NAME=nobl9
 BINARY=terraform-provider-${NAME}
 VERSION=0.1.4
```

Comment:
can you please also update the version in the example to be 0.2.0? I guess with CompositeSLOs we will release 0.2